src/aws_encryption_sdk/internal/crypto/authentication.py

```diff
@@ -58,15 +58,20 @@ class Signer(_PrehashingAuthenticator):
     """
 
     @classmethod
-    def from_key_bytes(cls, algorithm, key_bytes):
+    def from_key_bytes(cls, algorithm, key_bytes, encoding=serialization.Encoding.DER):
```

Comment:
not that it matters but how come the default is der?

Reply:
Check the before change -- this was only DER before. Now it's "DER unless you say otherwise"